zuul.d/edpm_multinode.yaml

```diff
@@ -210,6 +210,26 @@
     run:
       - ci/playbooks/edpm/run.yml
 
+- job:
+    name: podified-multinode-edpm-ironic-nobuild-tagged-crc
+    parent: cifmw-podified-multinode-edpm-base-crc
+    vars:
+      cifmw_extras:
+        - '@scenarios/centos-9/ci.yml'
+        - '@scenarios/centos-9/multinode-ci.yml'
+        - '@scenarios/centos-9/ceph_backends.yml'
+    run:
+      - ci/playbooks/e2e-run.yml
+    irrelevant-files:
+      - ^ci_framework/roles/.*_build
```

Comment:
this `ci_framework` folder doesn't exists anymore, so it would be `^roles/....`
see https://github.com/openstack-k8s-operators/ci-framework/commit/1f12633169ebabb6382d9f1ce0d726a39d23e6d2

Reply:
done